apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.ts

```diff
@@ -219,7 +219,7 @@ router.get(
         prompts,
         question: res.locals.question,
         richTextEditorEnabled,
-        variantId: typeof req.query?.variant_id === 'string' ? req.query?.variant_id : undefined,
+        variantId: typeof req.query.variant_id === 'string' ? req.query.variant_id : undefined,
```

Comment:
Some other code does `req.query.variant_id ?? null`. Not sure if one is preferred. This seems slightly worse.